pipelines/fast-test.yml

```diff
@@ -0,0 +1,216 @@
+# To reduce debug cost, steps are sorted differently on each platform,
```

Comment:
can't get the point, are these test cases in this file duplicated with other pipelines?

Reply:
This file is basically `azure-pipelines.yml`. I'm trying to do some refactor and don't want to block other PRs during development. When refactor is done, I will replace the old pipeline with this one.